packages/meta.ol

```diff
@@ -42,8 +41,7 @@ type IntRefinement {
 }
 
 /// A long basic type.
-type LongBasicType {
-	long
+type LongBasicType: long {
```

Comment:
This and the other *BasicType are not mistakes. I actually need a subnode called 'long' or something like that. Maybe we can use an annotation to generate something with a different name in Java?

Reply:
Java reserves void, int, enum, long, and double, so we'd need different names.

Reply:
would is... e.g. `isInt` make sense?

Reply:
Yes. :-)